src/main/java/roomescape/exeption/GlobalExeptionHandler.java

```diff
@@ -0,0 +1,14 @@
+package roomescape.exeption;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+public class GlobalExeptionHandler {
+
+    @ExceptionHandler(MethodArgumentTypeMismatchException.class)
+    public ResponseEntity<String> handleTypeMismatchException(MethodArgumentTypeMismatchException ex) {
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("");
```

Comment:
```suggestion
        return ResponseEntity.badRequest().body("예외 내용");
```
builder 패턴을 이용하는게 권장된다고 해요

Reply:
넵!